============================= test session starts ==============================
platform linux -- Python 3.12.6, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/runner/work/hw2/hw2
plugins: cov-5.0.0
collected 0 items / 1 error

==================================== ERRORS ====================================
________________________ ERROR collecting test_sort.py _________________________
ImportError while importing test module '/home/runner/work/hw2/hw2/test_sort.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.12.6/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_sort.py:6: in <module>
    from src.hw2_debugging import merge_sort
src/hw2_debugging.py:4: in <module>
    import rand
E   ModuleNotFoundError: No module named 'rand'

---------- coverage: platform linux, python 3.12.6-final-0 -----------
Name                   Stmts   Miss  Cover
------------------------------------------
src/hw2_debugging.py      22     21     5%
src/rand.py                6      6     0%
------------------------------------------
TOTAL                     28     27     4%
Coverage XML written to file post_traces/coverage.xml

=========================== short test summary info ============================
ERROR test_sort.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.18s ===============================
